qt/scientific_interfaces/Muon/ALCBaselineModellingView.h

```diff
@@ -59,6 +66,9 @@ private slots:
   void sectionsContextMenu(const QPoint &widgetPoint);
 
 private:
+  ALCBaselineModellingPresenter *m_presenter;
```

Comment:
Thanks for adding the interface class for the presenter! This can now be a `IALCBaselineModellingPresenter`, and also the parameter to the `subscribePresenter` method can be a `IALCBaselineModellingPresenter`